packages/utilities/components/compose.ts

```diff
@@ -0,0 +1,5 @@
+function compose(...f) {
```

Comment:
📐 Please type the input parameters.

Reply:
P.S. You should use overloading to properly type this functions for set number of input functions.

```ts
export function compose<A, B, C>(b: (v: B) => C, a: (v: A) => B): (a: A) => C
export function compose<A, B, C, D>(c: (v: C) => D, b: (v: B) => C, a: (v: A) => B): (a: A) => D
export function compose<A, B, C, D, E>(d: (v: D) => E, c: (v: C) => D, b: (v: B) => C, a: (v: A) => B): (a: A) => E
// ... 

export function compose(...f) {
//...
}
```

Reply:
done